### PR TITLE
Improve side citation

### LINKED ISSUE #246: Print view style improvements
- [x] Jacob: The author citation should appear whether the print settings are 1, 2, 3, 4 or more columns
- [x] Diego: Determine the number of characters that can appear in the citation on one vertical line on a “Letter” sized page. This number of characters should work for Legal and A4 as well, since they are longer. 
- [x] Diego: If the citation goes longer than that number of characters, then transform it so say “FIRST WORD et al.”
- [ ] truncate after one person when too long

### PR DESCRIPTION
**Pull request by Danble**
#### Relevant Issue
Closes #246 
#### Summarize what changed in this PR (for developers)
I created a new function for count and fix the citation side text.
I also added a dependency to handle special unicode characters 
#### Summarize changes in this PR (for public-facing changelog)
If the text becomes too long, it will only consider the first author and will add et al.
#### How can the changes be tested? Please also provide applicable links using preview deployments once they are available (will require editing this message once the preview deployment is ready).
http://localhost:3041/test-004/entries/print
https://living-dictionaries-git-improve-side-citation-livingtongues.vercel.app/test-004/entries/print

### CODE REVIEW

File: packages/site/src/routes/[dictionaryId]/entries/print/CitationClipped.svelte
Comment (jacob-8) [inline packages/site/src/routes/[dictionaryId]/entries/print/CitationClipped.svelte:7]: This component's only purpose is to receive names and a maxLength property. Can you please rewrite this PR to instead just add a function with tests that you can use in the +page.svelte file? At present it's very difficult to know what all this logic will do without starting a server and going to pages manually. This sort of thing is best served by a simple function with tests covering each use case. The stripHTMLTags is a good example to learn from. Take the function you have here in this component, put it in a typescript, and add enough tests to cover all the use cases.

**Review by jacob-8 — CHANGES_REQUESTED** [review]

**Comment by jacob-8** [comment]
@Danble Please review #248 and merge, then update this branch from main before continuing.

File: packages/site/src/routes/[dictionaryId]/entries/print/CitationClipped.svelte
Comment (Danble) [reply packages/site/src/routes/[dictionaryId]/entries/print/CitationClipped.svelte:7]: @jacob-8 That was the first thing I thought: create a function, but how can I get the text inside an HTML element? I could try to add an id to the div element but I don't know when to call the `document.getElementById` function.

**Review by jacob-8 — APPROVED** [review]
@Danble ready to merge if Anna and Greg are happy with the truncation behavior as I shared with them in the group thread.

**Comment by jacob-8** [comment]
@Danble this branch didn't get updated with the final decision to just use everything before the first divider (`,` `and` or `&`) + `et al` whether it is only a last name or a whole persons name written without being inverted. Can you please create a new PR with that adjustment - you'll want to start by spinning up Vitest and working until the tests reflect the desired change.

**Pull request merged**
